### DISCUSSION TITLE
Can we have a way where we can add weeks through something similar to the freeplay song list?

### DISCUSSION THREAD

**Discussion by SAOarmyVID** [KadeArchive/Kade-Engine#877] [Ideas]
I was thinking that maybe we could add weeks through a text file similar to the "freeplaySonglist.txt" file in the data folder. I even made and example of the config file layout that could be used to set up the week layers and animation. Thank you for reading this, and have a good one.
[weekConfigExample.txt](https://github.com/KadeDev/Kade-Engine/files/6665706/weekConfigExample.txt)

**Comment by isakube** [comment]
not how weeks work. at least, weeks specifically. This works better for Stages.

**Reply by Subsonic-Vaporeon** [reply]
@prokube your right, but their may be a possible way to add the weeks songs through a .txt file

**Reply by Subsonic-Vaporeon** [reply]
as long as they exist on Haxe already

**Comment by Lucky-56** [comment]
I could try adding this, it would make week modding a bit easier.

**Reply by SAOarmyVID** [reply]
If you could that would be absolutely amazing!

**Comment by puyoxyz** [comment]
no point to this, you're gonna be recompiling anyways so haxe knows the new assets exist, so you might as well just do it in the code

**Reply by Lucky-56** [reply]
yeah true

**Reply by SAOarmyVID** [reply]
The reason why is that some people can't install Haxe or get the tools necessary, this is more of a "Quality of Life" feature that would make modding much easier. And no, you wouldn't need to recompile unless you adding new mechanics or options.

**Reply by puyoxyz** [reply]
> The reason why is that some people can't install Haxe or get the tools necessary, this is more of a "Quality of Life" feature that would make modding much easier. And no, you wouldn't need to recompile unless you adding new mechanics or options.

YES YOU DO NEED TO RECOMPILE!!!! if you add an asset that wasn't there when compiling then HAXE DOESN'T KNOW IT EXISTS and gives you an error about it not existing!!!!!!
